frontend/src/audios.html

```diff
@@ -40,7 +54,12 @@
     </div>
     <div class="my-3 flex items-center">
       <!-- Filter button -->
-      <button data-filter-toggle class="mr-4 inline-flex w-24 items-center space-x-1 rounded-md border-gray-300 bg-neutral-600 px-4 py-2 font-semibold text-white shadow-sm md:hidden">
+      <button
+        type="button"
```

Comment:
Is this `type` required? I just don't know.

Reply:
type removed.